Mathlib/Tactic/ToAdditive.lean

```diff
@@ -374,6 +374,15 @@ def copyAttributes (src tgt : Name) : CoreM Unit := do
     (attrKind := AttributeKind.global)
     (prio := 1000) |>.run'
 
+/-- foo -/
+def copyInstanceAttribute (src tgt : Name) : CoreM Unit := do
+  if (← isInstance src) then
+    addInstance tgt AttributeKind.global 1000 |>.run'
```

Comment:
Ideally this would also copy the priority, can we add a todo note here for this?

Reply:
This is true and it is not totally trivial, because there are some nice features instances still missing in Core. But I will have a look at that later.